networking/verifying-connectivity-endpoint.adoc

```diff
@@ -11,5 +11,49 @@ By reviewing the results of the health checks, you can diagnose connection probl
 
 include::modules/nw-pod-network-connectivity-checks.adoc[leveloffset=+1]
 include::modules/nw-pod-network-connectivity-implementation.adoc[leveloffset=+1]
+
+[id="{context}-pod-placement"]
+== Connectivity check source and target placement
+
+:FeatureName: Custom configuration for connectivity check pod placement
+include::snippets/technology-preview.adoc[]
+
+You can configure the nodes on which network connectivity sources and targets run with a node selector. Additionally, you can specify permissible _tolerations_ for source and target pods. The configuration is defined in the singleton `cluster` custom resource of the `Network` API in the `config.openshift.io/v1` API group.
+
+Pod scheduling occurs after you have updated the configuration. Therefore, you must apply node labels that you intend to use in your selectors before updating the configuration. Labels applied after updating your network connectivity check pod placement are ignored.
+
+Refer to the default configuration in the following YAML:
+
+.Default configuration for connectivity source and target pods
```

Comment:
This section is a little confusing, currently this is the default value on a fresh 4.16 cluster:
```
    networkDiagnostics:
      mode: ""
      sourcePlacement: {}
      targetPlacement: {}
```

`mode: ""` - The user doesn't care about the mode (currently defaults to `All`)
`sourcePlacement: {}` - Currently defaults to any linux worker node - nodeSelector is set to `"kubernetes.io/os": "linux"` and there are no additional tolerations.
`targetPlacement: {}` -  Currently defaults to all linux nodes - nodeSelector is set to `"kubernetes.io/os": "linux"` and it tolerates all taints.

In general empty values mean that the CNO will decide the default values and there is a chance they will change in the future.